genomic_data_service/__init__.py

```diff
@@ -16,6 +16,9 @@ def is_web_app():
 
 if "GENOMIC_DATA_SERVICE_SETTINGS" in environ:
     app.config.from_envvar("GENOMIC_DATA_SERVICE_SETTINGS")
+elif "DOCKER" in environ:
+    print("[CONFIG] Defaulting to development_docker config")
+    app.config.from_pyfile("../config/development_docker.cfg")
 else:
     print("[CONFIG] Defaulting to development config")
     app.config.from_pyfile("../config/development.cfg")
```

Comment:
If we always set this variable either in Makefile or in docker-compose files, I think all we need is a check here that the variable is set and an raise appropriate exception otherwise?